.circleci/config.yml

```diff
@@ -153,8 +153,11 @@ commands:
       - run:
           name: Deploying to k8s cluster for service << parameters.k8s_svc_name >>
           command: |
+            git clone https://github.com/binary-com/devops-ci-scripts
+            cd ./devops-ci-scripts/k8s-build_tools
+            chmod +x release.sh
             echo $CA_CRT | base64 --decode > ca.crt
-            kubectl --server=${KUBE_SERVER} --certificate-authority=ca.crt --token=$SERVICEACCOUNT_TOKEN set image deployment/<< parameters.k8s_svc_name >> << parameters.k8s_svc_name >>=${DOCKHUB_ORGANISATION}/deriv-com:${CIRCLE_SHA1}
+            ./release.sh deriv-com ${CIRCLE_SHA1}
```

Comment:
where are we specifying the namespace for new-staging?

Reply:
In the Circleci environment variables